lib/propolis/src/block/mod.rs

```diff
@@ -57,6 +57,9 @@ pub type BlockPayload = dyn Any + Send + Sync + 'static;
 
 /// Block device operation request
 pub struct Request {
+    /// A device specific value identifying this request.
+    id: u32,
+
```

Comment:
Why not leave this as part of the payload?  Then NVMe could use struct members rather than having to pack it into a u32

Reply:
done